src/coordinates.cpp

```diff
@@ -6,6 +6,7 @@ double Coordinates::distanceTo(Coordinates point) const {
     double lat1 = this->latitude, lat2 = point.latitude;
     double lon1 = this->longitude, lon2 = point.longitude;
 
+    if ((lat1 == 0 && lon1 == 0) || (lat2 == 0 && lon2 == 0)) return -1;
```

Comment:
Why is this needed?

Reply:
So if there aren't coordinates set, it can identify and raise the appropriate error message (for the shipping graph)

Reply:
What if the coordinates are set and they are set to 0? Wouldn't that influence the other graphs?

Reply:
That's on you, that set the default to (0,0) 😭 But It's a huge coincidence if exists a point exactly on (0,0)

Reply:
Ok, I just checked there is not